### ISSUE TITLE
Property 'data' does not exist on type 'typeof import("discord-hybrid-sharding")'.

### ISSUE THREAD

**Issue by Mr-KayJayDee** [meister03/discord-hybrid-sharding#52]
**Error**
Property 'data' does not exist on type 'typeof import("discord-hybrid-sharding")'.

**Code**
```js
import Cluster from 'discord-hybrid-sharding';
import initialize from './initializer';

const Client = new Amandine({
    shards: Cluster.data.SHARD_LIST,
    shardCount: Cluster.data.TOTAL_SHARDS
});
```

**Package.json**
```json
{
  "name": "amandine-ts",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "build": "rm -rf dist && tsc",
    "start": "node dist/Cluster.js",
    "watch": "rm -rf dist/database/schemas && cp -r src/database/schemas dist/database/schemas && tsc --watch",
    "dev": "tsc && node dist/cluster.js"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@tsconfig/node16": "^1.0.2",
    "@types/figlet": "^1.5.4",
    "@types/i18next-fs-backend": "^1.1.2",
    "@types/node": "^16.11.41",
    "@types/superagent": "^4.1.15",
    "eslint": "^8.18.0",
    "typescript": "^4.7.4"
  },
  "dependencies": {
    "@discordjs/rest": "^0.4.1",
    "chartjs-node-canvas": "^4.1.6",
    "date-and-time": "^2.3.1",
    "discord-api-types": "^0.33.0",
    "discord-hybrid-sharding": "^1.6.2",
    "discord.js": "^13.7.0",
    "discord.js-light": "^4.7.0",
    "dotenv": "^16.0.0",
    "figlet": "^1.5.2",
    "i18next": "^21.8.10",
    "i18next-fs-backend": "^1.1.4",
    "ioredis": "^5.0.4",
    "mysql2": "^2.3.3",
    "stripe": "^9.11.0",
    "superagent": "^8.0.0"
  }
}
```

**Additional context**
Just updated to the latest version of discord-hybrid-sharding and got this error, working great before the update.

**Comment by meister03** [comment]
Use `Cluster.Client.getInfo()` as temporary solution..

**Comment by Mr-KayJayDee** [comment]
Okay, i tried updating to 1.6.4. Seems to be working with my code i sent. Maybe you have fixed it ?

Thanks for the help

**Comment by real2two** [comment]
I'm having the same problem as well. I tried using `Cluster.Client.getInfo()` but that's `undefined` as well. The problem only occurs to me when I use type "module".

**Comment by meister03** [comment]
Which version are you on, changed the handling on 1.6.7

**Comment by Mr-KayJayDee** [comment]
This is weird but it works

![Screenshot 2022-07-22 at 10 07 22](https://user-images.githubusercontent.com/46600662/180393864-7bb13092-50c5-4cf5-91a7-cacf9324f9ea.png)

**Comment by real2two** [comment]
(nevermind, my fault. I was starting my bot on the wrong file.)

**Comment by meister03** [comment]
Solved on `1.7.1` with `Client.Cluster.getInfo()`
